scripts/bin/base_acl_copp_svc.sh

```diff
@@ -13,8 +13,7 @@
 # See the Apache Version 2.0 License for specific language governing
 # permissions and limitations under the License.
 
-export LD_LIBRARY_PATH=/usr/lib/opx:/usr/lib/x86_64-linux-gnu:/usr/lib:/lib
-export PYTHONPATH=/usr/lib/opx  
+. /etc/opx/opx-environment.sh
 
```

Comment:
Spoke with Howard about this change. The change allows the script to be callable, even if it is used in .service file only today.